oxidation/crates/parsec_client_types/src/local_manifest.rs

```diff
@@ -324,6 +324,10 @@ impl LocalFileManifest {
         reference.version = remote_manifest.version;
         reference == *remote_manifest
     }
+
+    pub fn set_single_block(&mut self, block: u64, new_chunk: Chunk) {
+        self.blocks[block as usize] = vec![new_chunk];
```

Comment:
This is not safe :smile: match the get and return Err ? Or performance ?

Reply:
> match the get and return Err ?

Good catch :+1: 

> Or performance ?

Definitely not!

Reply:
[43d478f](https://github.com/Scille/parsec-cloud/pull/2334/commits/43d478f5f4a52687b6fa36e6aa441ba1c54be17e)